presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -219,6 +219,13 @@ Accessors
 
     Returns the cardinality of the collection of interior rings of a polygon.
 
+.. function:: line_locate_point(LineString, Point) -> double
```

Comment:
why not `ST_LineLocatePoint` too keep the convention across all the geospatial functions?

Reply:
@arhimondr ST_ prefix is reserved for functions which are part of SQL/MM Part 3 specification. Additional functions follow Presto naming scheme to avoid creating a false impression that these functions are part of the standard.